### PR TITLE
Add category info

### PR DESCRIPTION
**Pull request by bramkragten**
Adds endpoint for getting info for a category id.

### CODE REVIEW

File: spotipy/client.py [hunk 1]
```diff
@@ -1453,8 +1453,25 @@ def new_releases(self, country=None, limit=20, offset=0):
...
+            Parameters:
+                - category_id - The Spotify category ID for the category.
+
+                - country - An ISO 3166-1 alpha-2 country code.
+                - locale - The desired language, consisting of an ISO 639
+                  language code and an ISO 3166-1 alpha-2 country code, joined
+                  by an underscore.
+        """
+        return self._get(
+            "browse/categories" + category_id,
```
Comment (stephanebruckert) [inline spotipy/client.py:1468]: does it need a slash between these two bits?

**Comment by stephanebruckert** [comment]
Thanks for this. If you can, please write a quick test https://github.com/plamere/spotipy/blob/master/tests/integration/test_user_endpoints.py#L293

It requires a fresh account and a bit of set up, so don't worry too much about it if it doesn't work, I will check and fix it

File: spotipy/client.py (hunk 1 above)
Comment (bramkragten) [reply spotipy/client.py]: Yes!

**Comment by stephanebruckert** [comment]
Thanks, merging! I should be releasing 2.14.0 during the weekend

**Pull request merged**